README.md

```diff
@@ -2,3 +2,23 @@
 
 This repository contains the DDEV remote config definitions used by DDEV. For
 more information, see [DDEV's documentation](https://ddev.readthedocs.io/en/latest/developers/remote-config).
+
+## Contributing
+
+### Style guide
+
+This repository uses [Prettier](https://prettier.io/docs/en/) to maintain consistent formatting.
+You can manually run Prettier on this repository (requires Node.js).
+
+1. Clone or fork this project locally.
+1. Install node dependencies:
+
+   ```shell
+   npm install
+   ```
+
+1. Run the following command to attempt to auto-fix issues:
```

Comment:
I realize this is being picky, but if Markdown’s goal is to be human-readable it’s worth having sequential numbers out of principle:

```suggestion
1. Clone or fork this project locally.
2. Install node dependencies:

   ```shell
   npm install
   ```

3. Run the following command to attempt to auto-fix issues:
```

Reply:
> if Markdown’s goal is to be human-readable it’s worth having sequential numbers out of principle

Somebody actually agrees with me! I never thought anybody did!

Reply:
@rfay [John Gruber agrees with you](https://daringfireball.net/projects/markdown/), which should count for a lot.

> The overriding design goal for Markdown’s formatting syntax is to make it as readable as possible. The idea is that a Markdown-formatted document should be publishable as-is, as plain text, without looking like it’s been marked up with tags or formatting instructions.

Reply:
It sure makes maintaining markdown harder. But... you should be able to read it!

Reply:
> I don't understand really why anything matters besides the remote-config.jsonc

> I turn Prettier off in VS Code when I’m editing Markdown

I'm lost. You want to include markdown files in the linting process now?

Reply:
Sorry. I guess Matt was responding to earlier version, didn't realize that whole section had been removed, and then we went off on a tangent about markdown numbers. Nothing is wrong.

Reply:
Nothing is wrong except that Matt was saying use 1, 2, 3 in the README instead of 1, 1, 1. Not about linting.

Reply:
Oh yikes, sorry if I pointed at obsolete content! I _thought_ I was reviewing Markdown that had been added to the readme—which as you said had nothing to do with the linting itself.